modules/git/command.go

```diff
@@ -77,7 +83,11 @@ func (c *Command) RunInDirTimeoutEnvFullPipeline(env []string, timeout time.Dura
 	defer cancel()
 
 	cmd := exec.CommandContext(ctx, c.name, c.args...)
-	cmd.Env = env
+	if env == nil {
+		cmd.Env = append(os.Environ(), fmt.Sprintf("LC_ALL=%s", DefaultLCALL))
+	} else {
+		cmd.Env = append(env, fmt.Sprintf("LC_ALL=%s", DefaultLCALL))
```

Comment:
In this case `env` is expected to include `os.Environ()`?

Reply:
If `env` is `nil` then `cmd.Env` will default to `os.Environ()`. If it's not `nil` then the upstream caller decides.

Reply:
> 
> 
> If `env` is `nil` then `cmd.Env` will default to `os.Environ()`. If it's not `nil` then the upstream caller decides.

I think that's no longer true, because `append()` will add LC_ALL and an `env==nil` will become `env=[]string{"LC_ALL=....."}`, so there will not be a default anymore.

Reply:
see line 86... :stuck_out_tongue:

Reply:
> 
> 
> see line 86... 😛

🤦‍♂ I really need to sleep more.